reasoner/planner/ReasonerPlanPrinter.java

```diff
@@ -147,18 +147,18 @@ private Set<RuleMode> triggeredRuleModes(Concludable concludable, Set<Variable>
             builder.computeIfAbsent(new Pair<>(labels.get(callMode.conjunction), callMode.mode), labelMode -> new HashSet<>()).add(callMode.conjunction);
         });
 
-        Set<RuleMode> triggeredRuleModes = new HashSet<>();
-        builder.forEach((labelMode, conjunctions) -> triggeredRuleModes.add(new RuleMode(labelMode.first(), labelMode.second(), conjunctions)));
-        return triggeredRuleModes;
+        Set<LabelGroup> triggeredLabelGroups = new HashSet<>();
+        builder.forEach((labelMode, conjunctions) -> triggeredLabelGroups.add(new LabelGroup(labelMode.first(), labelMode.second(), conjunctions)));
+        return triggeredLabelGroups;
     }
 
-    private static class RuleMode {
+    private static class LabelGroup {
```

Comment:
Hmm I'd expect a `LabelGroup` to contain a group of Labels. Here it appears there's one group of conjunctions for one label/mode?
What about `CallGroup`?